wiki/People/Users_with_unique_titles/th.md

```diff
@@ -0,0 +1,345 @@
+| ![][flag_NL] [statementreply](https://osu.ppy.sh/users/126198) | Pro Tester |
+| ![][flag_KR] [SuddenDeath](https://osu.ppy.sh/users/6699923) | osu!mania Champion |
+| ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
+| ![][flag_KR] [T s u m i](https://osu.ppy.sh/users/4080520) | osu!catch Champion |
+| ![][flag_US] [t[-\_-t]](https://osu.ppy.sh/users/2644828) | osu! Champion |
+| ![][flag_KR] [Taeyang](https://osu.ppy.sh/users/2732340) | Elite Mapper |
+| ![][flag_US] [Thievley](https://osu.ppy.sh/users/4717672) | osu!artist |
+| ![][flag_RU] [tieff](https://osu.ppy.sh/users/89619) | Featured Artist |
+| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | osu! Champion |
+| ![][flag_KR] [Transcendence](https://osu.ppy.sh/users/903155) | osu!mania Champion |
+| ![][flag_JP] [uone](https://osu.ppy.sh/users/5321719) | osu!taiko Champion |
+| ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150) | osu! Champion |
+| ![][flag_CA] [VINXIS](https://osu.ppy.sh/users/4323406) | Featured Artist |
+| ![][flag_KR] [wonder5193](https://osu.ppy.sh/users/8474029) | osu!mania Champion |
+| ![][flag_US] [wonjae](https://osu.ppy.sh/users/5032045) | Elite Nominator |
+| ![][flag_US] [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist |
+| ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661) | Elite Mapper |
+| ![][flag_KR] [Yeoul](https://osu.ppy.sh/users/4698728) | osu!mania champion |
+| ![][flag_HM] [Zallius](https://osu.ppy.sh/users/55) | Boats |
+| ![][flag_DE] [Zetera](https://osu.ppy.sh/users/587737) | Medal Hunter |
+| ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768) | Elite Mapper |
+
+[flag_AT]: /wiki/shared/flag/AT.gif "Austria"
+[flag_AU]: /wiki/shared/flag/AU.gif "Australia"
+[flag_CA]: /wiki/shared/flag/CA.gif "Canada"
+[flag_CL]: /wiki/shared/flag/CL.gif "Chile"
+[flag_CN]: /wiki/shared/flag/CN.gif "China"
+[flag_DE]: /wiki/shared/flag/DE.gif "Germany"
+[flag_ES]: /wiki/shared/flag/ES.gif "Spain"
+[flag_FI]: /wiki/shared/flag/FI.gif "Finland"
+[flag_FR]: /wiki/shared/flag/FR.gif "France"
+[flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
+[flag_HM]: /wiki/shared/flag/HM.gif "Heard Island and McDonald Islands"
+[flag_ID]: /wiki/shared/flag/ID.gif "Indonesia"
+[flag_IT]: /wiki/shared/flag/IT.gif "Italy"
+[flag_JP]: /wiki/shared/flag/JP.gif "Japan"
+[flag_KR]: /wiki/shared/flag/KR.gif "South Korea"
+[flag_LY]: /wiki/shared/flag/LY.gif "Libya"
+[flag_MY]: /wiki/shared/flag/MY.gif "Malaysia"
+[flag_NL]: /wiki/shared/flag/NL.gif "Netherlands"
+[flag_NO]: /wiki/shared/flag/NO.gif "Norway"
+[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
+[flag_PL]: /wiki/shared/flag/PL.gif "Poland"
+[flag_RU]: /wiki/shared/flag/RU.gif "Russian Federation"
+[flag_SE]: /wiki/shared/flag/SE.gif "Sweden"
+[flag_SG]: /wiki/shared/flag/SG.gif "Singapore"
+[flag_SH]: /wiki/shared/flag/SH.gif "Saint Helena"
+[flag_TW]: /wiki/shared/flag/TW.gif "Taiwan"
+[flag_US]: /wiki/shared/flag/US.gif "United States"
```

Comment:
you need to translate the tooltips as well

Reply:
Translated in the lastest commit.